optimum/exporters/openvino/model_configs.py

```diff
@@ -592,6 +592,20 @@ class GptOssOpenVINOConfig(LlamaOpenVINOConfig):
 class ExaoneOpenVINOConfig(LlamaOpenVINOConfig):
     pass
 
+@register_in_tasks_manager(
+    "exaone4",
+    *[
+        "feature-extraction",
+        "feature-extraction-with-past",
+        "text-generation",
+        "text-generation-with-past",
+        "text-classification",
```

Comment:
do you validate tasks "feature-extraction", "text-classification" in the precommit?
Please check